tableschema/profile.py

```diff
@@ -35,14 +35,29 @@ def jsonschema(self):
         return self.__jsonschema
 
     def validate(self, descriptor):
+
+        # General validation
         if self.name != 'table-schema':
             return jsonschema.validate(descriptor, self.jsonschema)
+
+        # Table schema validation
+        errors = []
         validator = _TableSchemaValidator(
             self.jsonschema, format_checker=jsonschema.FormatChecker())
-        errors = list(validator.iter_errors(descriptor))
+        for error in validator.iter_errors(descriptor):
+            if isinstance(error, jsonschema.exceptions.ValidationError):
+                path = '/'.join(map(str, error.path))
+                schema_path = '/'.join(map(str, error.schema_path))
+                error = exceptions.ValidationError(
+                    'Descriptor validation error: %s '
+                    'at "%s" in descriptor and '
+                    'at "%s" in profile'
+                    % (error.message, path, schema_path))
```

Comment:
@amercader 
Related to ugly Python2's `u` prefixes. Should we here just remove it like `message = str(error.message).replace('u\'', '\'')`?